app/code/ZP/LoyaltyProgram/Setup/Patch/Data/AddBasicPrograms.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace ZP\LoyaltyProgram\Setup\Patch\Data;
+
+use Magento\Framework\Setup\Patch\DataPatchInterface;
+use ZP\LoyaltyProgram\Api\Data\LoyaltyProgramInterface;
+use ZP\LoyaltyProgram\Api\Data\LoyaltyProgramInterfaceFactory;
+use ZP\LoyaltyProgram\Model\ResourceModel\LoyaltyProgram as LoyaltyProgramResource;
+use Magento\Framework\Exception\AlreadyExistsException;
+
+class AddBasicPrograms implements DataPatchInterface
+{
+    public function __construct(
+        private LoyaltyProgramInterfaceFactory $loyaltyProgramFactory,
+        private LoyaltyProgramResource $loyaltyProgramResource
+    ) {}
+
+    /**
+     * @return DataPatchInterface
+     * @throws AlreadyExistsException
+     */
+    public function apply(): DataPatchInterface
+    {
+        /** @var LoyaltyProgramInterface $programMinimum */
+        $programMinimum = $this->loyaltyProgramFactory->create();
+        $programMinimum->setProgramName('Program Minimum')
+            ->setDescription('Some Description for Program Minimum')
+            ->setConditionsSerialized('Some Conditions for Program Minimum');
+        $this->loyaltyProgramResource->save($programMinimum);
+
+        /** @var LoyaltyProgramInterface $programMaximum */
+        $programMaximum = $this->loyaltyProgramFactory->create();
+        $programMaximum->setProgramName('Program Maximum')
```

Comment:
what if you set program ID 1 here and try to save it?
And also define program id = 2 as a public constant PROGRAM_MAX ?